src/time.rs

```diff
@@ -256,3 +256,33 @@ pub fn clock_getcpuclockid(pid: Pid) -> Result<ClockId> {
         Err(Error::Sys(Errno::from_i32(ret)))
     }
 }
+
+pub type ClockNanosleepFlags = crate::sys::timerfd::TimerSetTimeFlags;
```

Comment:
I'm curious what maintainers think of this type alias. I worry it might not be forwards compatible.

Reply:
Never mind - that worked by chance, only because libc::TIMER_ABSTIME == 1 and libc::TFD_TIMER_ABSTIME == 1. I fixed it by creating an entirely new type.